ts/test/session/unit/utils/RingBuffer_test.ts

```diff
@@ -0,0 +1,106 @@
+// tslint:disable: no-implicit-dependencies max-func-body-length no-unused-expression no-require-imports no-var-requires
+
+import chai from 'chai';
+import { RingBuffer } from '../../../../session/utils/RingBuffer';
+
+const { expect } = chai;
+
+describe('RingBuffer Utils', () => {
+  it('gets created with right capacity', () => {
+    const ring = new RingBuffer<number>(5000);
+    expect(ring.getCapacity()).to.equal(5000);
+    expect(ring.getLength()).to.equal(0);
+    expect(ring.has(0)).to.equal(false, '4 should not be there');
```

Comment:
Shouldn't this be "0 should not be there"?

Reply:
yep